apps/web/playwright/organization/organization-settings.e2e.ts

```diff
@@ -66,7 +66,7 @@ async function verifyRobotsMetaTag({ page, orgSlug, urls, expectedContent }: Ver
   await doOnOrgDomain({ orgSlug, page }, async ({ page, goToUrlWithErrorHandling }) => {
     for (const relativeUrl of urls) {
       const { url } = await goToUrlWithErrorHandling(relativeUrl);
-      const metaTag = await page.locator('head > meta[name="robots"]');
+      const metaTag = await page.locator('head > meta[name="robots"]').first();
```

Comment:
Because we didn't completely deprecate `NextSeo` yet (we can completely deprecate once we are 100% done with migration), we can have two metadata tags with `robots` in the html.